astrodb_utils/tests/test_sources.py

```diff
@@ -88,14 +88,36 @@ def test_find_source_in_db(db):
     )
     assert len(search_result) == 0
 
-    search_result = find_source_in_db(db,"LHS 2924")
+    search_result = find_source_in_db(db, "LHS 2924")
     assert search_result[0] == "LHS 2924"
 
-    search_result = find_source_in_db(db,"LHS 292", fuzzy=False)
+    search_result = find_source_in_db(db, "LHS 292", fuzzy=False)
     assert len(search_result) == 0
 
-    search_result = find_source_in_db(db,"LHS 292", fuzzy=True)
-    assert search_result[0] == "LHS 2924"  # This is wrong and a result of fuzzy matching
+    search_result = find_source_in_db(db, "LHS 292", fuzzy=True)
+    assert (
+        search_result[0] == "LHS 2924"
+    )  # This is wrong and a result of fuzzy matching
+
+
+def test_find_source_in_db_with_coords(db):
```

Comment:
Suggested test addition: have the real coordinates of some source but then trim it down to 4-5 digits and see if it fails.